README.md

```diff
@@ -1,29 +1,48 @@
 # YJDH services
 
-## kesaseteli
+## kesaseteli employer
 
-YJDH-Kesäseteli
+YJDH-Kesäseteli service for employers to fulfill employee applications
 
-### kesaseteli Development with Docker
+### kesaseteli-employer Development with Docker
 
-1. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+1. Run `yarn` to install necessary packages
 
-2. Run `yarn kesaseteli` or, if you want to rebuild, then `yarn kesaseteli --build`
+2. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+
+3. Run `yarn employer` or, if you want to rebuild, then `yarn employer --build`
 
 The Frontend is now running at [localhost:3000](https://localhost:3000)
 The backend is now running at [localhost:8000](https://localhost:8000)
 
 3. If services fail to get up, `yarn clean` might help.
 
+## kesaseteli youth
+
+YJDH-Kesäseteli service for young people to send kesäseteli applications
+
+### kesaseteli-youth Development with Docker
+
+1. Run `yarn` to install necessary packages
+
+2. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+
+3. Run `yarn youth` or, if you want to rebuild, then `yarn youth --build`
+
+The Frontend is now running at [localhost:3100](https://localhost:3000)
+The backend is now running at [localhost:8000](https://localhost:8000)
```

Comment:
Now the two last rows are shown actually on a single row when viewing the README.md through a markdown viewer and not on different rows.

To fix this they could e.g. be put under the "3." section:
```
3. Run `yarn youth` or, if you want to rebuild, then `yarn youth --build`
   - The Frontend is now running at [localhost:3100](https://localhost:3000)
   - The backend is now running at [localhost:8000](https://localhost:8000)
```

Reply:
fixed